vault/core_metrics.go

```diff
@@ -136,6 +136,11 @@ func (c *Core) metricsLoop(stopCh chan struct{}) {
 // TokenStore; there is one per method because an additional level of abstraction
 // seems confusing.
 func (c *Core) tokenGaugeCollector(ctx context.Context) ([]metricsutil.GaugeLabelValues, error) {
+	if c.IsDRSecondary() {
+		// there is no expiration manager on DR Secondaries
+		return []metricsutil.GaugeLabelValues{}, nil
```

Comment:
This makes sense to me!

Reply:
another option (for these 4 token gauge collectors) is to add some flag to `metricsInit` in `emitMetrics`